mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/DynamicLoadProviderResourceDefinition.java

```diff
@@ -53,26 +58,37 @@
     public static final PathElement LEGACY_PATH = PathElement.pathElement("dynamic-load-provider", "configuration");
     public static final PathElement PATH = PathElement.pathElement("load-provider", "dynamic");
 
-    enum Attribute implements org.jboss.as.clustering.controller.Attribute {
+    enum Attribute implements org.jboss.as.clustering.controller.Attribute, UnaryOperator<SimpleAttributeDefinitionBuilder> {
         DECAY("decay", ModelType.DOUBLE, new ModelNode((double) DynamicLoadBalanceFactorProvider.DEFAULT_DECAY_FACTOR)),
         HISTORY("history", ModelType.INT, new ModelNode(DynamicLoadBalanceFactorProvider.DEFAULT_HISTORY)),
+        INITIAL_LOAD("initial-load", ModelType.INT, new ModelNode(0)) {
```

Comment:
Is 0 a good default here?  In a typical setup how long will it take for a node to get a normal share of load?

Reply:
I am open to discussion – this pre-populates with full load. The formula for these numbers to be out of the history "lookback" is = history * status interval = which is 90 seconds. However, practically, the since the default decay factor is 2, this has minimal effect much quicker.

Reply:
That sounds fine; thanks.